terraform/modules/front_door/cloudfront.tf

```diff
@@ -59,6 +60,12 @@ resource "aws_cloudfront_distribution" "this" {
     }
   }
 
+  logging_config {
+    bucket          = aws_s3_bucket.cloudfront_access_logs.bucket_domain_name
+    include_cookies = false
```

Comment:
Do you agree there isn't value in recording cookie information in the access logs?

If enabled, Cloudfront will log all cookies in the access logs regardless of it they are usually forward to the origin or not. I believe we use a `_data_collector_session` cookie in the app, and in the browser it looks like this stores an encrypted value about the session. I didn't personally see much value in logging this and other analytics cookies but may have missed something!

Reply:
Sounds reasonable to me, but Rachael might have more knowledge to bring to the question!

Reply:
Would the cookies ever give us useful information if we needed to look through the logs? I'm not immediately sure of the answer.

Reply:
Had a chat with Kat about this

Looks like the `_data_collection_session` cookie holds your sign-in info and it's value is sensitive (because someone who had access to where this info is stored could then sign in as you, by just setting it as their own cookie), so not sure it's a good idea to record it. It also stores info like filters selected on the page which I don't think is useful. I believe the Cloudwatch Access Logs will most likely be viewed if there is a security concern, and we need to verify what requests have come in and from which Client IPs for example, so I'm leaning strongly on it isn't necessary and risky to store if we do.